src/manifest.lisp

```diff
@@ -49,6 +62,19 @@ does not exist."
            (return-from manifest-get-by-name matches)))))
 
 
+(defun manifest-update ()
+  "Upgrade the Qi manifest by checking out the latest version from git."
+  (cond ((probe-file (fad:merge-pathnames-as-file +manifest-directory+ "manifest.lisp"))
+         (format t "~%---> Upgrading Qi manifest~%")
+         (run-git-command "fetch origin" +manifest-directory+)
+         (run-git-command "rebase origin/master" +manifest-directory+))
```

Comment:
> Any local changes will surely have a merge conflict

Not necessarily.  Homebrew uses the stash/rebase/pop approach and it seems to work.  But either is fine with me.